ci-operator/step-registry/hypershift/mce/power/OWNERS

```diff
@@ -1,13 +1,9 @@
 approvers:
-- csrwng
-- enxebre
-- sjenning
 - mkumatag
 - dharaneeshvrd
+- Shilpa-Gokul
 options: {}
 reviewers:
-- csrwng
-- enxebre
-- sjenning
 - mkumatag
 - dharaneeshvrd
+- Shilpa-Gokul
```

Comment:
please add the newline in the end, applies in the other places as well

Reply:
fixed